Extension/tests/css-filter/test-css-filter.js

```diff
@@ -519,35 +483,35 @@ QUnit.test("Extended Css Build CssHits", function (assert) {
 
     var shadowDomPrefix = adguard.utils.browser.isShadowDomSupported() ? "::content " : "";
 
-    var rule = new adguard.rules.CssFilterRule("adguard.com##.sponsored");
-    var genericRule = new adguard.rules.CssFilterRule("##.banner");
-    var extendedCssRule = new adguard.rules.CssFilterRule("adguard.com##.sponsored[-ext-contains=test]");
+    var rule = new adguard.rules.CssFilterRule("adguard.com##.sponsored", 1);
+    var genericRule = new adguard.rules.CssFilterRule("##.banner", 2);
+    var extendedCssRule = new adguard.rules.CssFilterRule("adguard.com##.sponsored[-ext-contains=test]", 1);
     var filter = new adguard.rules.CssFilter([rule, genericRule, extendedCssRule]);
 
     var selectors, css, extendedCss, commonCss;
 
-    selectors = filter.buildCssHits("adguard.com", 'hit-prefix');
+    selectors = filter.buildCssHits("adguard.com");
     css = selectors.css;
     extendedCss = selectors.extendedCss;
     commonCss = filter.buildCssHits(null).css;
     assert.equal(commonCss.length, 1);
-    assert.equal(commonCss[0].trim(), shadowDomPrefix + ".banner { display: none!important; background-image: url('hit-prefix/elemhidehit.png#%3B%23%23.banner') !important;}");
+    assert.equal(commonCss[0].trim(), shadowDomPrefix + ".banner { display: none!important; content: 'adguard2%3B%23%23.banner' !important;}");
     assert.equal(css.length, 2);
-    assert.equal(css[0].trim(), shadowDomPrefix + ".banner { display: none!important; background-image: url('hit-prefix/elemhidehit.png#%3B%23%23.banner') !important;}");
-    assert.equal(css[1].trim(), shadowDomPrefix + ".sponsored { display: none!important; background-image: url('hit-prefix/elemhidehit.png#%3Badguard.com%23%23.sponsored') !important;}");
+    assert.equal(css[0].trim(), shadowDomPrefix + ".banner { display: none!important; content: 'adguard2%3B%23%23.banner' !important;}");
+    assert.equal(css[1].trim(), shadowDomPrefix + ".sponsored { display: none!important; content: 'adguard1%3Badguard.com%23%23.sponsored' !important;}");
     assert.equal(extendedCss.length, 1);
-    assert.equal(extendedCss[0].trim(), ".sponsored[-ext-contains=test] { display: none!important; background-image: url('hit-prefix/elemhidehit.png#%3Badguard.com%23%23.sponsored%5B-ext-contains%3Dtest%5D') !important;}");
+    assert.equal(extendedCss[0].trim(), ".sponsored[-ext-contains=test] { display: none!important; content: 'adguard1%3Badguard.com%23%23.sponsored%5B-ext-contains%3Dtest%5D' !important;}");
 
-    selectors = filter.buildCssHits("adguard.com", 'hit-prefix', true);
+    selectors = filter.buildCssHits("adguard.com", true);
```

Comment:
I suppose we need a way of unit-testing the whole feature:

1. Building proper CSS with hits counter
2. Collecting rule hits by the content script
3. Processing of these stats by the CssHitsCounter